cirq/sim/state_vector_simulator_test.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2019 The Cirq Developers
```

Comment:
nit: 2020

Reply:
This is a rename, think it should stay 2019?